api/v1/server/handlers/v1/workflow-runs/trigger.go

```diff
@@ -122,6 +122,16 @@ func (t *V1WorkflowRunsService) V1WorkflowRunCreate(ctx echo.Context, request ge
 		return nil, err
 	}
 
+	if request.Body.AdditionalMetadata != nil {
+		correlationId, ok := (*request.Body.AdditionalMetadata)["correlationId"].(string)
```

Comment:
Probably safer to do:

```go
if correlationIdInterface, exists := (*request.Body.AdditionalMetadata)["correlationId"]; exists {
        if correlationId, ok := correlationIdInterface.(string); ok {

        }
    }
```

For all of these - since additional metadata is a `map[string]interface{}` and some old SDKs were technically able to send JSON as the interface value